rdr_service/tools/tool_libs/ptsc_retention_metrics.py

```diff
@@ -0,0 +1,71 @@
+#! /bin/env python
+#
+# Template for RDR tool python program.
+#
+import argparse
+import logging
+import sys
+
+from rdr_service.offline.retention_eligible_import import import_retention_eligible_metrics_file
+from rdr_service.services.system_utils import setup_logging, setup_i18n
+from rdr_service.tools.tool_libs import GCPProcessContext, GCPEnvConfigObject
+
+_logger = logging.getLogger("rdr_logger")
+
+# Tool_cmd and tool_desc name are required.
+# Remember to add/update bash completion in 'tool_lib/tools.bash'
+tool_cmd = "ptsc-retention"
+tool_desc = "load ptsc retention eligible metrics into rdr"
+
+
+class PTSCRetentionMetricsClass(object):
+    def __init__(self, args, gcp_env: GCPEnvConfigObject):
+        """
+        :param args: command line arguments.
+        :param gcp_env: gcp environment information, see: gcp_initialize().
+        """
+        self.args = args
+        self.gcp_env = gcp_env
+
+    def run(self):
+        """
+        Main program process
+        :return: Exit code value
+        """
+        self.gcp_env.activate_sql_proxy()
+
+        # TODO: Glob the bucket and get the latest file or get file date from command line arg.
+        task_payload = {
+            'bucket': 'ptsc-metrics-all-of-us-rdr-prod',
+            'file_path': 'ptsc-metrics-all-of-us-rdr-prod/2021-07-26_retention_and_ubr_data.csv',
+            'upload_date': '2021-07-26 11:24:21'
```

Comment:
Is this intended to still be hardcoded for now?

Reply:
Yes, for now.  It can be updated in the future to grab the latest bucket file or take one from a command line arg.

Reply:
I changed my mind and removed the bucket and blob name.